lib/http_health_check/probes/sidekiq.rb

```diff
@@ -14,7 +14,7 @@ def initialize(sidekiq: ::Sidekiq)
 
       def probe(_env)
         @sidekiq_module.redis do |conn|
-          conn.setex(meta[:redis_key], TTL_SEC, MAGIC_NUMBER)
```

Comment:
SETEX is now deprecated: https://redis.io/commands/setex/
This call works fine within Redis and RedisClient ([starting from Sidekiq 7](https://github.com/sidekiq/sidekiq/blob/main/docs/7.0-Upgrade.md#redis-client)) API